packages/medusa/src/services/draft-order.ts

```diff
@@ -274,10 +274,22 @@ class DraftOrderService extends TransactionBaseService {
           }
         }
 
-        const createdCart = await cartServiceTx.create({
+        let createdCart = await cartServiceTx.create({
           type: CartType.DRAFT_ORDER,
           ...rawCart,
         })
+        createdCart = await cartServiceTx.retrieve(createdCart.id, {
+          relations: [
+            "billing_address",
+            "customer",
+            "discounts",
+            "discounts.rule",
+            "gift_cards",
+            "items",
+            "region",
+            "shipping_address",
+          ],
+        })
```

Comment:
Loading relations based on all the Cart properties that can be populated by the [CartService.create](https://github.com/medusajs/medusa/blob/33bb6044c800620ab97183680d8b9702b43f63d4/packages/medusa/src/services/cart.ts#L400-L520) operation.